tests/test_recipient_validation.py

```diff
@@ -327,83 +323,6 @@ def test_validate_email_address_raises_for_invalid(email_address):
     assert str(e.value) == 'Not a valid email address'
 
 
-@pytest.mark.parametrize('column', [
-    'address_line_1', 'AddressLine1',
-    'postcode', 'Postcode'
-])
-@pytest.mark.parametrize('contents', [
-    '', ' ', None
-])
-def test_validate_address_raises_for_missing_required_columns(column, contents):
-    with pytest.raises(InvalidAddressError) as e:
-        validate_recipient(contents, 'letter', column=column)
-    assert 'Missing' == str(e.value)
-
-
-@pytest.mark.parametrize('column', [
-    'address_line_3',
-    'address_line_4',
-    'address_line_5',
-    'address_line_6',
-])
-def test_validate_address_doesnt_raise_for_missing_optional_columns(column):
-    assert validate_recipient('', 'letter', column=column) == ''
-
-
-def test_validate_address_raises_for_wrong_column():
-    with pytest.raises(TypeError):
-        validate_recipient('any', 'letter', column='email address')
-
-
-@pytest.mark.parametrize('column', [
-    'address_line_1',
-    'address_line_2',
-    'address_line_3',
-    'address_line_4',
-    'address_line_5',
-    'address_line_6',
-])
-def test_validate_address_allows_any_non_empty_value_for_non_postcode(column):
-    assert validate_recipient('any', 'letter', column=column) == 'any'
-
-
-@pytest.mark.parametrize('column', [
-    'address_line_1',
-    'address_line_2',
-    'address_line_3',
-    'address_line_4',
-    'address_line_5',
-    'address_line_6',
-])
-def test_non_ascii_address_line_is_fine_for_non_postcode(column):
```

Comment:
Do we wish to retain any checks that non ascii characters are fine in general in all our placeholders or is this just fine to assume?

Reply:
We’re not doing anything unusual with encoding or downgrading so I think we can assume that it should behave like any other string.